packages/sources/wrapped/src/endpoint/deposits.ts

```diff
@@ -0,0 +1,46 @@
+import { Requester, Validator, AdapterError } from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig, InputParameters, AxiosResponse } from '@chainlink/types'
+
+export const supportedEndpoints = ['deposits']
+
+export type Addresses = {
+  address: string
+}
+
+const customError = (data: any) => data.Response === 'Error'
+
+export const inputParameters: InputParameters = {
+  symbol: false,
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, _, config) => {
+  const validator = new Validator(request, inputParameters)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const symbol = validator.validated.data.symbol || 'ETH'
+  const url = `/deposits`
+
+  const options = { ...config.api, url }
+  const response = await Requester.request(options, customError)
```

Comment:
If you pass in the response structure as generic here the response will be typed, which can be helpful for logic that happens before the result is returned. 

```suggestion
  const response = await Requester.request<Addresses[]>(options, customError)
```